hbt/config/variables.py

```diff
@@ -288,23 +267,378 @@ def add_variables(config: od.Config) -> None:
+        aux={"inputs": build_dijet.inputs},
+        binning=(30, 0, 300),
+        unit="GeV",
+        x_title=r"$m_{jj}$",
+    )
+    add_variable(
+        config,
+        name="dijet_pt",
+        expression=partial(build_dijet, which="pt"),
+        aux={"inputs": build_dijet.inputs},
+        binning=(40, 0, 200),
+        unit="GeV",
+        x_title=r"$p_{T,jj}$",
+    )
+    add_variable(
+        config,
+        name="dijet_eta",
+        expression=partial(build_dijet, which="eta"),
+        aux={"inputs": build_dijet.inputs},
+        binning=(50, -5, 5),
+        x_title=r"$\eta_{jj}$",
+    )
+    add_variable(
+        config,
+        name="dijet_phi",
+        expression=partial(build_dijet, which="phi"),
+        aux={"inputs": build_dijet.inputs},
+        binning=(34, -3.3, 3.3),
+        x_title=r"$\phi_{jj}$",
+    )
+    add_variable(
+        config,
+        name="dijet_dr",
+        expression=partial(build_dijet, which="dr"),
+        aux={"inputs": build_dijet.inputs},
+        binning=(30, 0, 6),
+        x_title=r"$\Delta R_{jj}$",
+    )
+
+    # dilepton variables
+    add_variable(
+        config,
+        name="dilep_energy",
+        expression=partial(build_dilep, which="energy"),
+        aux={"inputs": build_dilep.inputs},
+        binning=(40, 40, 300),
+        unit="GeV",
+        x_title=r"$E_{ll}$",
+    )
     add_variable(
         config,
         name="dilep_mass",
-        expression=dilep_mass_test,
-        aux={
-            "inputs": ["{Electron,Muon,Tau}.{pt,eta,phi,mass}"],
-        },
+        expression=partial(build_dilep, which="mass"),
+        aux={"inputs": build_dilep.inputs},
         binning=(40, 40, 120),
         unit="GeV",
         x_title=r"$m_{ll}$",
     )
+    add_variable(
+        config,
+        name="dilep_pt",
+        expression=partial(build_dilep, which="pt"),
+        aux={"inputs": build_dilep.inputs},
+        binning=(40, 0, 200),
+        unit="GeV",
+        x_title=r"$p_{T,ll}$",
+    )
+    add_variable(
+        config,
+        name="dilep_eta",
+        expression=partial(build_dilep, which="eta"),
+        aux={"inputs": build_dilep.inputs},
+        binning=(50, -5, 5),
+        unit="GeV",
+        x_title=r"$\eta_{ll}$",
+    )
+    add_variable(
+        config,
+        name="dilep_phi",
+        expression=partial(build_dilep, which="phi"),
+        aux={"inputs": build_dilep.inputs},
+        binning=(34, -3.3, 3.3),
+        unit="GeV",
+        x_title=r"$\phi_{ll}$",
+    )
+    add_variable(
+        config,
+        name="dilep_dr",
+        expression=partial(build_dilep, which="dr"),
+        aux={"inputs": build_dilep.inputs},
+        binning=(30, 0, 6),
+        x_title=r"$\Delta R_{ll}$",
+    )
+
+    # hh variables
+    add_variable(
+        config,
+        name="hh_energy",
+        expression=partial(build_hh, which="energy"),
+        aux={"inputs": build_hh.inputs},
+        binning=(40, 100, 800),
```

Comment:
```suggestion
        binning=(35, 100, 800),
```
bins with size 20 better than 17.5